src/SourceLink.Common/build/Microsoft.SourceLink.Common.props

```diff
@@ -7,14 +7,13 @@
 
   <PropertyGroup>
     <!--
-      Do not query source control manager for any information when building in the IDE or for Live Unit Testing.
-      Features that need this information will be disabled if false.
+      Used to suppress querying source control and features that use the information (e.g. git commit SHA).
     -->
-    <EnableSourceControlManagerQueries Condition="'$(EnableSourceControlManagerQueries)' == '' and '$(DesignTimeBuild)' != 'true' and '$(BuildingForLiveUnitTesting)' != 'true'">true</EnableSourceControlManagerQueries>
```

Comment:
The LUT condition still seems reasonable--is it affected by the same set of problems? Is there signoff from the LUT team?

Reply:
If there is a test that reads AssemblyInformationalVersion it would behave differently when running LUT. I think it better if they are the same between DTB and command line build.

Reply:
The LUT team agreed over IM.